src/celeritas/em/model/CoulombScatteringModel.cc

```diff
@@ -60,6 +60,10 @@ CoulombScatteringModel::CoulombScatteringModel(ActionId id,
         = imported_.energy_grid_bounds(data_.ids.electron, MaterialId{0});
 
     // Check that the bounds are the same for all particles/materials
+    // \todo This is only expected when using Coulomb scattering with the
```

Comment:
the `\todo` command is only parsed as part of a doxygen block, so this would need to be annotated with `/*! ...`